resources/lang/en/general.php

```diff
@@ -253,7 +253,7 @@
     'signature'             => 'Signature',
     'signed_off_by'         => 'Signed Off By',
     'skin'       			=> 'Skin',
-    'webhook_msg_note'        => 'A :app message will be sent',
```

Comment:
I think this one is okay? We pass the `:app` variable to the translation

Reply:
Actually, no I'm wrong. We don't pass it there - we should, but we don't currently I think

Reply:
You're right. It wasn't being passed in so I started the process of passing it in but the message looked off to me when you have "general" selected as your webhook type
![example message](https://user-images.githubusercontent.com/1141514/227302178-a0ab3dd3-b23d-4b58-984b-85ca1887638b.png)